site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/FileUploader.js

```diff
@@ -19,16 +21,24 @@ const FileUploader = ({ dropzoneParams, maxFileSize, name, currentFiles, handleD
                 return (
                     <div {...getRootProps()}>
                         <input {...inpProps} />
-                        <label className="helptext mt-0">
+                        <label className="helptext mt-0 mb-0">
                             {`Optional. Max attachments size:`} {humanReadableBytes(maxFileSize, false)}
                         </label>
+
                         {currentFiles.length > 0
                             &&
                             <FileTable
+                                className="rel-mt-1"
                                 handleDelete={handleDelete}
                                 filesList={currentFiles}
                             />
                         }
+                        
+                        {errorMessage &&
+                            <Message negative>
```

Comment:
@zzacharo [comment](https://github.com/zenodo/zenodo-rdm/pull/89#discussion_r1022763072):

> I think for consistency with the other instances, I would put the error summary at the top of the form...

Reply:
I think there is a misunderstanding here, as we don't have an error summary for all the fields, only highlighted field-errors.